packages/rehype/src/core.ts

```diff
@@ -45,15 +52,22 @@ export interface RehypeShikiExtraOptions {
   parseMetaString?: (
     metaString: string,
     node: Element,
-    tree: Root
+    tree: Root,
   ) => Record<string, any> | undefined | null
 
+  /**
+   * Highlight inline code blocks
+   *
+   * @default false
+   */
+  inline?: boolean
```

Comment:
```suggestion
  inline?: false | 'curly-colon'
```

Something like this to indicate the style? In case we might have multiple ones in the future as https://github.com/shikijs/shiki/issues/660#issuecomment-2069110522

Reply:
Brilliant idea, I've updated the docs too